modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -22,186 +22,187 @@ ContentDashboardAdminDisplayContext contentDashboardAdminDisplayContext = (Conte
 									%>
 
 										<clay:label
-											displayType="secondary"
-											large="<%= true %>"
-										>
-											<clay:label-item-expand><%= assetCategory.getTitle(locale) %></clay:label-item-expand>
-										</clay:label>
+											label="<%= StringUtil.toUpperCase(status.getLabel()) %>"
+											style="<%= status.getStyle() %>"
+										/>
 
 									<%
 									}
 									%>
 
 								</liferay-ui:search-container-column-text>
 
-							<%
-							}
-							%>
+								<%
+								for (AssetVocabulary assetVocabulary : contentDashboardAdminDisplayContext.getAssetVocabularies()) {
+								%>
+
+									<liferay-ui:search-container-column-text
+										cssClass="table-cell-expand-smaller"
+										name="<%= assetVocabulary.getTitle(locale) %>"
+									>
 
-							<liferay-ui:search-container-column-date
-								name="modified-date"
-								value="<%= contentDashboardItem.getModifiedDate() %>"
-							/>
+										<%
+										for (AssetCategory assetCategory : (List<AssetCategory>)contentDashboardItem.getAssetCategories(assetVocabulary.getVocabularyId())) {
+										%>
+
+											<clay:label
+												displayType="secondary"
+												large="<%= true %>"
+											>
+												<clay:label-item-expand><%= assetCategory.getTitle(locale) %></clay:label-item-expand>
+											</clay:label>
+
+										<%
+										}
+										%>
 
-							<liferay-ui:search-container-column-text>
-								<clay:dropdown-actions
-									dropdownItems="<%= contentDashboardAdminDisplayContext.getDropdownItems(contentDashboardItem) %>"
+									</liferay-ui:search-container-column-text>
+
+								<%
+								}
+								%>
+
+								<liferay-ui:search-container-column-date
+									name="modified-date"
+									value="<%= contentDashboardItem.getModifiedDate() %>"
 								/>
-							</liferay-ui:search-container-column-text>
-						</liferay-ui:search-container-row>
 
-						<liferay-ui:search-iterator
-							markupView="lexicon"
-						/>
-					</liferay-ui:search-container>
-				</aui:form>
+								<liferay-ui:search-container-column-text>
+									<%
+										Map<String, Object> additionalProps = HashMapBuilder.<String, Object>put(
+											"sidebarContainerSelector", ".sidebar-container"
+										).build();
+									%>
+
+									<clay:dropdown-actions
+										additionalProps="<%= additionalProps %>"
+										dropdownItems="<%= contentDashboardAdminDisplayContext.getDropdownItems(contentDashboardItem) %>"
+										propsTransformer="js/ActionsComponentPropsTransformer"
+									/>
+								</liferay-ui:search-container-column-text>
+							</liferay-ui:search-container-row>
+
+							<liferay-ui:search-iterator
+								markupView="lexicon"
+							/>
+						</liferay-ui:search-container>
+					</aui:form>
+				</div>
 			</div>
-		</clay:container-fluid>
+		</div>
+	</clay:container-fluid>
+
+	<div class="sidebar-container">
```

Comment:
Why not render this in the component and avoid having to pass a `sibebarContainerSelector`?

Reply:
I need to specify a selector in which render the component, isn't it?
```
const sidebarRef = ReactDOM.render(
```

Reply:
So, maybe it would be better to provide a container that already exists and add your own thingie in there rather than having users create an empty `div` in a jsp and pass you a selector to it?

Reply:
Are we talking in a general way to make a pattern? If so, you might want to render the component in a different place than I, that's why I'm passing the selector.

Or maybe I'm not getting it 😅

Reply:
Let's add the requirement that you cannot add markup to that jsp :)

Reply:
We don't add markup for a modal container or a toast container... there are patterns we use to do that, so I'm saying maybe some of that applies here too.

Reply:
Modal and toast containers are rendered out of your html structure, this is not. I can hardcode it to render it into the row
```
<clay:row
	cssClass="no-gutters"
        id="foo"
>
```
```
const sidebarRef = ReactDOM.render(
		<BridgeComponent 
			bridgedComponent={Sidebar}
			onClose={hideSidebar}
		/>,
		document.querySelector('#foo')
	);
```

but I'm trying to do it in a way we can stablish a pattern...

Reply:
This looks like a simpler pattern since it doesn't require creating an empty div placeholder.

![just-sayin](https://user-images.githubusercontent.com/905006/87554114-02b43980-c6b4-11ea-8495-6f470382708a.gif)

 I know from feedback that many of our existing patterns that bridge java-jsp-js are quite hard to use because they require making synced changes in many places, so trying to get to something that is easier to wire

Reply:
Well, I can't actually do what I just said 'cause I'd be removing the rest of the html inside of the row (so, everything)

Reply:
Do we still need this?